airbyte-integrations/bases/source-acceptance-test/unit_tests/test_spec_unit.py

```diff
@@ -23,64 +23,77 @@
 #
 
 import io
-import os
-from pathlib import Path
 
 import docker
+import pytest
 from source_acceptance_test.utils import ConnectorRunner
 
 
+def create_dockerfile(text, tag, tmp_path):
+    client = docker.from_env()
+    fileobj = io.BytesIO()
+    fileobj.write(text)
+
+    fileobj.seek(0)
+    args = {
```

Comment:
please use kwargs in rare cases only, 
you can't validate name and type of arguments if you call a function like this, making much harder to spot issues in IDE